packages/dev/materials/src/custom/customMaterial.ts

```diff
@@ -198,6 +182,19 @@ export class CustomMaterial extends StandardMaterial {
 
         this.FragmentShader = Effect.ShadersStore["defaultPixelShader"];
         this.VertexShader = Effect.ShadersStore["defaultVertexShader"];
+
+        CustomMaterial.ShaderIndexer++;
+        this._createdShaderName = "custom_" + CustomMaterial.ShaderIndexer;
+    }
+
+    protected _afterBind(mesh?: Mesh | undefined, effect?: Nullable<Effect>): void {
+        if (!effect) {
+            return;
+        }
+        this.AttachAfterBind(mesh, effect);
+        try {
+            super._afterBind(mesh, effect);
+        } catch (e) {}

```

Comment:
might be a naive question, but usually when i see an empty catch i wonder - in what case is it ok for this to fail? shouldn't it be logged?